packages/xchain-thorchain/genMsgDeposit.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
Can you put a note to the README about this script, pls?

Reply:
I added a note here: https://github.com/xchainjs/xchainjs-lib/pull/501/files#diff-c903bffebd5e89b3f2f3b6fc7813460bc9b3edd1dd386262631c0732712b4cf9R68 - let me know if you want something else

Reply:
Looks good, thx!